bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java

```diff
@@ -1664,6 +1685,9 @@ public void initialize() {
         snapshotUri = getCorrectUrlFormat(cameraConfig.getSnapshotUrl());
         mjpegUri = getCorrectUrlFormat(cameraConfig.getMjpegUrl());
         rtspUri = cameraConfig.getFfmpegInput();
+        if (cameraConfig.getFfmpegOutput().isEmpty()) {
+            cameraConfig.setFfmpegOutput(OpenHAB.getUserDataFolder() + "/cameras/" + this.thing.getUID().getId() + "/");
```

Comment:
I think it would be a good approach to put the files into a sub folder named exactly as your binding as you might not the only one in this folder.

Reply:
DONE: used ipcamera instead of cameras.